spec/support/test_helper.rb

```diff
@@ -13,6 +13,28 @@ def legacy_encrypt_private_key(private_key, password)
   encrypted_private_key
 end
 
+def prepare_transferred_encryptable(bob, alice, encryption_algorithm)
```

Comment:
can't we use a Fabricator for this?

so in the test we just could call: Fabricate(:transferred_encryptable, sender: bob, recipient: alice)

then use logic from  EncryptableTransfer for creating it ?

would find this way more clean than the helper here